modules/sbo-naming-strategies.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-naming-strategies_{context}"]
+= Naming strategies
+
+This section describes the naming strategies, which are available only for the `binding.operators.coreos.com` API group.
```

Comment:
```suggestion
Naming strategies are available only for the `binding.operators.coreos.com` API group.
```
Remove `This section describes`. It doesn't really add anything meaningful, and anyway the title of the section implies what the content will be.